research/src/components/component-coverage.js

```diff
@@ -17,13 +17,26 @@ const ComponentCoverage = ({ component }) => {
       <ul style={{ marginLeft: 0 }}>
         {_.map(withDifferentNamesUniq, (component, i) => {
           const components = withDifferentNamesGrouped[component.name]
-          const sourceNames = _.map(components, 'sourceName').join(', ')
 
           return (
             <li key={component.name} style={{ listStyleType: 'none' }}>
               {component.name}
               {' - '}
-              <small style={{ opacity: 0.75 }}>{sourceNames}</small>
+              <small style={{ opacity: 0.75 }}>
+                {components.map((component, i, arr) => (
+                  <>
+                    <a
+                      key={component.sourceName}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href={component.url}
+                    >
+                      {component.sourceName}
+                    </a>
```

Comment:
Here's the actual change, we now link the design system for component names back to the design system page for that component:

![image](https://user-images.githubusercontent.com/5067638/76123421-6d8a9100-5fad-11ea-8248-f51322638022.png)